tests/test_oml/test_datasets/test_audio.py

```diff
@@ -0,0 +1,95 @@
+import random
+
+import pandas as pd
+import pytest
+
+from oml.const import PATHS_COLUMN, START_TIME_COLUMN
+from oml.utils import get_mock_audios_dataset
+from tests.test_oml.test_datasets.test_datasets import (
+    check_base,
+    check_labeled,
+    check_query_gallery,
+)
+
+
+@pytest.fixture(scope="module")
+def df() -> pd.DataFrame:
+    df_train, df_val = get_mock_audios_dataset(global_paths=True)
+    return pd.concat([df_train, df_val])
+
+
+@pytest.mark.needs_optional_dependency
+@pytest.mark.parametrize("num_channels", [1, 2])
+def test_downmix(df: pd.DataFrame, num_channels: int) -> None:
+    from oml.datasets.audios import AudioBaseDataset
+
+    dataset = AudioBaseDataset(df[PATHS_COLUMN].tolist(), num_channels=num_channels)
+    for item in dataset:
+        audio = item[dataset.input_tensors_key]
+        assert audio.shape[0] <= num_channels, f"Audio channels {audio.shape[0]} exceed specified {num_channels}"
```

Comment:
why `<=` but not a `!=` ?

Reply:
yeah, this was dumb from me to use `num_channels`, because there is usually (99%) only two or one channel, and users mostly downmix them into mono audios and leave the same (very rare), so I changed it from `num_channels` to `is_mono`, so it would be more logically right

Reply:
okay, let's keep this approach

But `is_mono: bool = DEFAULT_IS_MONO` sounds like a flag indicating if an audio is in mono format or not.
I suggest `convert_to_mono: bool = CONVERT_TO_MONO_DEFAULT` which is better semantically from my point of view.